development/wasm-filter-plugins.md

```diff
@@ -0,0 +1,162 @@
+# WASM Filter Plugins
+
+Fluent Bit currently supports integration of wasm plugins built as wasm/wasi objects for input and filter plugins only. The interface for the WASM filter plugins is currently under development but is functional.
```

Comment:
@cosmo0920 can you introduce WASM a little first maybe?

Also a sentence per line is easier to review.

Reply:
I added in https://github.com/fluent/fluent-bit-docs/pull/942/commits/95ee5c271d79d5ffddae09578bf0191945c453e9.